test/playwright/test_ux.py

```diff
@@ -168,12 +204,19 @@ def _existing_environment_interactions(page, env_name, time_to_build_env=3*60*10
     """
     # edit existing environment throught the YAML editor
     page.get_by_role("button", name=env_name).click()
+    if screenshot:
+        page.keyboard.press("PageUp")  # ensure we are at the top of the page
+        page.screenshot(path="test-results/edit-env.png")
     page.get_by_role("button", name="Edit").click()
+    if screenshot:
+        page.screenshot(path="test-results/switch-to-yaml.png", clip={'x': 280, 'y': 385, 'width': 985, 'height': 75})
+        page.keyboard.press("PageDown")  # ensure we are at the bottom of the page
+        page.screenshot(path="test-results/delete-env.png")
```

Comment:
We may need to pagedown (and clip?) to show the delete environemtn link, it's not currently captured

Ref: https://github.com/conda-incubator/conda-store/pull/731/files#diff-64e535b9b95a7d5c840aafc7725bc97e15ab6d2829699efa2f8415929af0ca83